bin/citrea/src/main.rs

```diff
@@ -241,13 +249,16 @@ where
             error!("Error: {}", e);
         }
     } else {
-        let rollup = CitreaRollupBlueprint::create_new_rollup(
+        let (mut rollup, rpc_methods) = CitreaRollupBlueprint::create_new_rollup(
             &rollup_blueprint,
             rt_genesis_paths,
             rollup_config,
         )
         .await
         .expect("Could not start full-node");
+
+        rollup.start_rpc_server(rpc_methods, None).await;
```

Comment:
Is it on purpose that we don't return errors and panic from FullNode::start_rpc_server?

Reply:
tbh, I don't even know